es-ds-docs/layouts/default.vue

```diff
@@ -1,24 +1,22 @@
 <template>
-    <div>
-        <div class="d-flex justify-content-center">
-            <div class="ds-side-nav d-none d-xl-block flex-shrink-0 p-100">
-                <ds-link-list />
-            </div>
-            <b-container class="pt-xl-100 mx-0">
-                <b-row class="mb-100">
-                    <b-col cols="12">
+    <div class="d-flex justify-content-center">
+        <b-container class="pt-100 mx-0">
```

Comment:
The `16px` margin was coming from the top nav before, so just making a note we'll need to remove `mb-100` from that element when it gets added back.

Reply:
Yeah we were using the default styling of Bootstrap Vue's navbar component before, which I'm not sure we should put effort into bringing forward into ESDS 3.0, as we have no Figma designs for it. Breadcrumbs and category pages may be enough to navigate on mobile, but once we put more thought into navs in general (and potentially rebuild the EsNavBar to be accessible), we may discover some better PrimeVue tools we can use for this.